src/components/NanoContract/NanoContractTransactionsListHeader.component.js

```diff
@@ -0,0 +1,265 @@
+  };
+
+  return (
+    <HathorHeader>
+      <HathorHeader.Central style={styles.headerCentral}>
+        <TouchableWithoutFeedback onPress={() => toggleShrank(!isShrank)}>
+          <View style={styles.headerWrapper}>
+            <HeaderIcon />
+            <Text style={[styles.headerTitle]}>{t`Nano Contract`}</Text>
+            {isShrank
+              && <HeaderShrank />}
+            {isExpanded()
+              && (
+              <HeaderExpanded
+                nc={nc}
+                address={address}
+                onEditAddress={onEditAddress}
+                onUnregisterNanoContract={onUnregisterNanoContract}
+              />
+              )}
+          </View>
+        </TouchableWithoutFeedback>
+        <SelectAddressModal
+          address={selectedAddress}
+          show={showSelectAddressModal}
+          onDismiss={toggleSelectAddressModal}
+          onSelectAddress={hookSelectAddress}
+        />
+        <UnregisterNanoContractModal
+          ncId={nc.ncId}
+          show={showUnregisterNanoContractModal}
+          onDismiss={toggleUnregisterNanoContractModal}
+        />
+      </HathorHeader.Central>
+    </HathorHeader>
+  )
+};
+
+const HeaderIcon = () => (
+  <View>
+    <NanoContractIcon type='fill' color={COLORS.white} />
+  </View>
+);
+
+const HeaderShrank = () => (
+  <ArrowDownIcon />
+);
+
+/**
+ * Presents all the information.
+ *
+ * @param {Object} props
+ * @param {Object} props.nc Nano Contract data
+ * @param {string} props.address Current address
+ * @param {() => void} props.onEditAddress Function called on address edition trigger
+ * @param {() => void} props.onUnregisterNanoContract Function called on unregister trigger
+ */
+const HeaderExpanded = ({ nc, address, onEditAddress, onUnregisterNanoContract }) => {
+  const baseExplorerUrl = useSelector((state) => state.networkSettings.explorerUrl);
+
+  const navigatesToExplorer = () => {
+    const txUrl = `transaction/${nc.ncId}`;
+    const explorerLink = combineURLs(baseExplorerUrl, txUrl);
+    Linking.openURL(explorerLink);
+  };
+
+  return (
+    <>
+      <View style={styles.wrapper}>
+        <InfoContainer>
+          <TextValue bold pb4>{getShortHash(nc.ncId, 7)}</TextValue>
+          <TextLabel>{t`Nano Contract ID`}</TextLabel>
+        </InfoContainer>
+        <InfoContainer>
+          <TextValue bold pb4>{nc.blueprintName}</TextValue>
+          <TextLabel>{t`Blueprint Name`}</TextLabel>
+        </InfoContainer>
+        <EditInfoContainer center onPress={onEditAddress}>
+          <TextValue bold pb4>{getShortContent(address, 7)}</TextValue>
+          <TextLabel>{t`Registered Address`}</TextLabel>
+        </EditInfoContainer>
+        <TwoActionsWrapper>
+          <PrimaryTextButton title={t`See status details`} onPress={navigatesToExplorer} />
+          <DenyTextButton title={t`Unregister contract`} onPress={onUnregisterNanoContract} />
+        </TwoActionsWrapper>
+      </View>
+      <ArrowUpIcon />
```

Comment:
Should this be inside the View?

Reply:
Let me try.

Reply:
No. Keep it this way. It needs to be in the edge.